### ISSUE TITLE
Issue: Unable to edit killboard

### ISSUE THREAD

**Issue by CokkocZateki** [evekb/evedev-kb#45]
Hi.

I have a issue where after installation i can access the ACP but cannot do any changes on the site. Whenever i click on a tickbox or try to change anything and click save it undoes the changes.

**Comment by Salvoxia** [comment]
Hi,

is this issue related to #46 ?
Could you please elaborate on what you have tried in what order and what the current state is?

Best Regards,
Salvoxia

**Comment by Noobforlife** [comment]
no it doesnt seem to be related to issue 46. i logged onto the admin page and whatever tickbox i click on and tried to save doesnt get applied. so im not able to do anychanges in the admin panel. I thought it may be the SSL key since its not loading all the content on the page.

**Comment by CokkocZateki** [comment]
no it doesnt seem to be related to issue 46. i logged onto the admin page and whatever tickbox i click on and tried to save doesnt get applied. so im not able to do anychanges in the admin panel. I thought it may be the SSL key since its not loading all the content on the page.

**Comment by Salvoxia** [comment]
Remote diagnosis is a bit difficult. 
You are running the killboard on SSL, so could you please check the following: When clicking the save button, monitor the the calls and make sure the URL the post request is sent to respects the HTTPS request scheme and does not simply run against http.

The only other instance of a similar problem I can remember is when the killboard was hosted at GoDaddy. They use(d) a PHP security module, blocking all POST/GET requests containing URLs as parameter values. That is the case for the Configuration Page in ACP, as well as some other Pages (zKB Fetch, IDFeed), but not for the Front Page, for example.

**Comment by CokkocZateki** [comment]
Hi.

Yes this SSL is from go daddy unfortunatly but the vps is hosted somewhere else(vps.net). when i click save my broswer does give me an message (Although the page is encypted, the information you have entered is to be sent over an unecryted connection and could easily be ready by a third party)

the post request still stays as HTTPS so no changes there

**Comment by Salvoxia** [comment]
Hi,

I assume there's nothing in the PHP error logs either? 
Frankly, I'm running out of ideas...

I can offer to do some hands-on debugging, though.

Regards,
Salvoxia

**Comment by CokkocZateki** [comment]
can you pm me or email me and if possible go thru this together?

**Comment by Salvoxia** [comment]
Re: [evedev-kb] Issue: Unable to edit killboard (#45)

Hi,

sure, what modus operandi did you have in mind?

Regards,Salvoxia

Hello CokkocZateki,

Saturday, December 12, 2015, 12:55:10 AM, you wrote:

can you pm me or email me and if possible go thru this together?

On Fri, Dec 11, 2015 at 4:03 PM, Salvoxia notifications@github.com wrote:

> Hi,
> 
> I assume there's nothing in the PHP error logs either?
> Frankly, I'm running out of ideas...
> 
> Regards,
> Salvoxia
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/evekb/evedev-kb/issues/45#issuecomment-163958006.
> 
> —
> Reply to this email directly or view it on GitHub.

## 

Best regards,
 Salvoxia                            mailto:Salvoxia@blindfish.info

**Comment by CokkocZateki** [comment]
no i dont find any errors in the PHP error logs :-(

**Comment by Salvoxia** [comment]
Re: [evedev-kb] Issue: Unable to edit killboard (#45)

Hi,

ok, as a last resort I can offer to do some hands-on debugging.
However, in order to this effectively I need access to your killboard's ACP and to the webspace the killboard is sitting on (via FTP). 
Most hosters let you create a second FTP account with separate access control, so that user would only have access to the killboard and nothing else. The EDK admin password can be easy enough by simple setting a new one in kbconfig.php. It will be encrypted by EDK  on the next login attempt.

It's on you to decide whether you like me to try that and if I'm trustworthy enough ;)

Regards,
Salvxoai

Monday, December 14, 2015, 11:11:17 AM, you wrote:

no i dont find any errors in the PHP error logs :-(

On Sun, Dec 13, 2015 at 2:58 PM, Salvoxia notifications@github.com wrote:

> Re: [evedev-kb] Issue: Unable to edit killboard (#45)
> 
> Hi,
> 
> sure, what modus operandi did you have in mind?
> 
> Regards,Salvoxia
> 
> Hello CokkocZateki,
> 
> Saturday, December 12, 2015, 12:55:10 AM, you wrote:
> 
> can you pm me or email me and if possible go thru this together?
> 
> On Fri, Dec 11, 2015 at 4:03 PM, Salvoxia notifications@github.com
> wrote:
> 
> > Hi,
> > 
> > I assume there's nothing in the PHP error logs either?
> > Frankly, I'm running out of ideas...
> > 
> > Regards,
> > Salvoxia
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/evekb/evedev-kb/issues/45#issuecomment-163958006.
> > 
> > —
> > Reply to this email directly or view it on GitHub.
> 
> ## 
> 
> Best regards,
> Salvoxia mailto:Salvoxia@blindfish.info
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/evekb/evedev-kb/issues/45#issuecomment-164261601.
> 
> —
> Reply to this email directly or view it on GitHub.

## 

Best regards,
 Salvoxia                            mailto:Salvoxia@blindfish.info

**Comment by Icy0ne** [comment]
I have just experienced this issue but my circumstances may be a little different.

The killboard was migrated to a new host and a new domain. A 301 redirect was in place from old to new domain. Everything looked fine until trying to save anything in the admin pages. After using the Chrome dev tools I saw the 301 redirect taking place and realised what was happening. I modified the cfg_kbhost and cfg_img entries the config table and saving worked again.
